properties/genre.rb

```diff
@@ -0,0 +1,13 @@
+require 'securerandom'
+
+class Genre
+  def initialize(name)
+    @id = SecureRandom.hex(5)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
Good job, but don't forget to add :

`item.add_genre(self)` in the `add_item` method